modules/test/playwright/pages/layout-admin-web/PagesAdminPage.ts

```diff
@@ -579,4 +579,16 @@ export class PagesAdminPage {
 
 		await this.page.getByLabel('Select All Items').waitFor();
 	}
+
+	async addCustomCSS(css: string) {
+		await this.page
+			.getByRole('radio', {name: 'Define a custom theme for'})
```

Comment:
Add `this.page
			.getByRole('radio', {name: 'Define a custom theme for'})` as a locator in the constructor. Replace other usages.